apps/juicebox_web/assets/js/videos/actions.js

```diff
@@ -29,12 +30,27 @@ function toggleExpanded() {
   }
 }
 
+function skipPlaying({ streamId }) {
+  console.log(streamId);
```

Comment:
Unexpected console statement.

[Read more about it here.](http://eslint.org/docs/rules/no-console)